app/jobs/get_usps_proofing_results_job.rb

```diff
@@ -305,7 +305,12 @@ def handle_expired_status_update(enrollment, response, response_message)
     end
 
     begin
-      send_deadline_passed_email(enrollment.user, enrollment) unless enrollment.deadline_passed_sent
+      unless enrollment.deadline_passed_sent
+        send_deadline_passed_email(
+          enrollment.user, enrollment,
+          'none'
```

Comment:
In the scenario where the user does not attempt to proof at a post office before the deadline passes, there is no visited location name because there is no visited location.  Hence, 'none.'

I also contemplated using inheritance and creating a child of `VerificationResultsEmailPresenter`, `DeadlinePassedEmailPresenter`, that wouldn't take `visited_location_name` as an argument for its initialize method.  Curious what other engineers think!

Reply:
I had mixed feeling about this. We are passing in 'none' to send_deadlin_passed_email and then again in the mailer. I thought maybe good to not pass it in- and just set the default to none in the mailer. I think I like this better because we stay consistent with passing in 3 ordered args. I think when we refactor this (usps job), I'd like to see us passing around hashes. I think it is easier when there are 3+ args.) I would not do that here in this PR- that requires a lot of changes so blows up scope here. All this to say- this is fine for me.

Reply:
I also went back and forth about this internally.  I decided to refactor to use keyword arguments, in part because of points made [here](https://github.com/18F/identity-idp/pull/11353/files#r1805061643).